CMakeLists.txt

```diff
@@ -270,6 +270,10 @@ if(GCC OR CLANG)
     set(C_CXX_FLAGS "${C_CXX_FLAGS} -Wimplicit-fallthrough")
   endif()
 
+  if(CMAKE_COMPILER_IS_GNUCXX AND CMAKE_CXX_COMPILER_VERSION VERSION_GREATER "5")
```

Comment:
Not sure if `AND CMAKE_CXX_COMPILER_VERSION VERSION_GREATER "5"`should be added? I don't see it in the Boringssl `CMakeLists.txt` and it didn't show up in my merge attempts.

Reply:
Blame-ing on this file shows it comes from this commit https://github.com/awslabs/aws-lc/commit/f22f88ecb1e2afb7a0efaa0da23bbaecdd3f850f by @bryce-shang

Reply:
The flag does not exist before g++-5.